### PR TITLE
Feat/display standardized ft images/i1800

### LINKED ISSUE #1800: Display standardized FT images (e.g. MIA)
It appears as though despite supporting API-based metadata for standardized FT attributes, we aren't showing token images yet, such as for MiamiCoin:

image

### PR DESCRIPTION
**Pull request by beguene**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/stacks-wallet-web/actions/runs/2415450778).

fixes #1800
ft5
ft4
ft3
ft2
ft1

It's almost done, I need to test a bit more and do some cleanup.
You can test the Popup using the test-app, there is 'Send Miamicoin' using Mainnet (to get the correct metadata)

cc/ @kyranjamie @fbwoolf @beguene @He1DAr

### CODE REVIEW

**Comment by markmhendrickson** [comment]
cc @whoabuddy @pstan26  =)

**Review by kyranjamie — COMMENTED** [review]
Nice one @beguene, wallet will look a lot better with this. 

Main thoughts are around other non-legit names; we probs want a lib similar to https://github.com/mathiasbynens/punycode.js

e.g: `STＸ`, `Stḁcks` and the many more, less distinguishable, similar chars.

File: src/app/common/token-utils.ts [hunk 1]
```diff
@@ -14,3 +15,30 @@ export function getFormattedAmount(amount: string) {
       }
     : { value: amount, isAbbreviated: false };
 }
+
+export function isIconUrl(iconString: string): boolean {
+  return /^https?:\/\//.test(iconString);
+}
```
Comment (kyranjamie) [inline src/app/common/token-utils.ts]: Could you instead use the utility `isValidUrl`?

**Review by fbwoolf — APPROVED** [review]
🎉

**Comment by markmhendrickson** [comment]
> Main thoughts are around other non-legit names; we probs want a lib similar to > https://github.com/mathiasbynens/punycode.js
> 
> e.g: STＸ, Stḁcks and the many more, less distinguishable, similar chars.

@kyranjamie to address

**Review by kyranjamie — APPROVED** [review]

**Comment by beguene** [comment]
FYI @Eshwari007 and @timstackblock this is ready for QA.
Please note that you should use mainnet to get the correct images

**Comment by Eshwari007** [comment]
> FYI @Eshwari007 and @timstackblock this is ready for QA. Please note that you should use mainnet to get the correct images

Will keep you posted upon testing @beguene

File: src/app/common/token-utils.ts [hunk 2]
```diff
@@ -14,3 +17,30 @@ export function getFormattedAmount(amount: string) {
       }
     : { value: amount, isAbbreviated: false };
 }
+
+export function isIconUrl(iconString: string): boolean {
+  return isValidUrl(iconString);
+}
```
Comment (kyranjamie) [inline src/app/common/token-utils.ts]: this fn doesn't really add/do anything, could use `isValidUrl` directly, or if you think renaming is valuable, could be

```
const isIconUrl = isValidUrl
```

**Pull request merged**